rmd_files/RNAseq_report.Rmd

```diff
@@ -3619,21 +3622,31 @@ if ( runArribaChunk ) {
 ##### Otherwise add empty columns expected from Aribba results
```

Comment:
Should we remove this comment as we are no longer adding empty columns for Arriba results?

Reply:
Good point